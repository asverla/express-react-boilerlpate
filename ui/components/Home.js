import React from 'react'
import DemoComponentLeft from './DemoComponentLeft'
import DemoComponentRight from './DemoComponentRight'
import s from '../styles/components/home.less'

class Home extends React.Component {
  render () {
    return (
      <div id='home' className={s.homeContainer}>
        <DemoComponentLeft id='demoLeft' />
        <DemoComponentRight id='demoRight' />
      </div>
    )
  }
}

export default Home
