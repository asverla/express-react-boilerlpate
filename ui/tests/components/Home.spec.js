/* eslint-disable no-unused-expressions */
import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import toJson from 'enzyme-to-json'

import Home from '../../components/Home'

Enzyme.configure({ adapter: new Adapter() })

describe('When Home component renders', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(<Home />)
  })

  it('should match the snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  afterEach(() => {
    wrapper.unmount()
  })
})
