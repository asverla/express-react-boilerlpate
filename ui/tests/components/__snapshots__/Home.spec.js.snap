// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When Home component renders should match the snapshot 1`] = `
<div
  className="homeContainer"
  id="home"
>
  <b
    id="demoLeft"
  />
  <b
    id="demoRight"
  />
</div>
`;
